pepper-apis/dsm-core/src/main/java/org/broadinstitute/dsm/db/dao/kit/KitDaoImpl.java

```diff
@@ -200,8 +198,8 @@ public Optional<ScanError> updateKitRequest(KitRequestShipping kitRequestShippin
             } catch (Exception ex) {
                 logger.error("Not able to update the kit for ddpLabel " + kitRequestShipping.getDdpLabel(), ex);
                 dbVals.resultValue = new ScanError(kitRequestShipping.getDdpLabel(),
-                        "Kit Label \"" + kitRequestShipping.getDdpLabel() + "\" was already scanned.\n"
-                                + UserErrorMessages.IF_QUESTIONS_CONTACT_DEVELOPER);
+                        String.format("Kit Label \"%s\" w was already scanned.\n"
```

Comment:
Seems like this is a user input error, yet we are logging at error level?

Reply:
Yes, this scan error is sent to the frontend and is displayed for the user

Reply:
You're correct! This is a user error most of the times, so basically no we don't necessarily need to log at this level, but changing this is not in the scope of this ticket. There are a lot of these instances in DSM, I think if we want to change that I'd say this class alone needs it's own ticket based on the number of changes and checks we have to make.